tests/automated_install.sh

```diff
@@ -6,6 +6,11 @@
 # simple few environment variables to tune booting for fast testing of Chef-BCPC
 # Run this script in the root of the git repository
 #
+# Pre-requisite software:
+# 1. curl
+# 2. virtualbox
+# 3. vagrant
+# 4. chefdk (w/ berkshelf)
```

Comment:
We want to be as lightweight as possible on the hypervisor host; please restrict to installing and running berks on the bootstrap node.